ui/admin/app/templates/scopes/scope/add-storage-policy/index.hbs

```diff
@@ -0,0 +1,26 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
```

Comment:
heads up the <Breadcrumbs::Item> component is missing for this path

Reply:
great catch thanks! -  added